src/main/kotlin/com/autonomousapps/internal/reason/ModuleAdviceExplainer.kt

```diff
@@ -0,0 +1,69 @@
+package com.autonomousapps.internal.reason
+
+import com.autonomousapps.internal.utils.Colors
+import com.autonomousapps.model.AndroidScore
+import com.autonomousapps.model.ProjectCoordinates
+import com.autonomousapps.tasks.ReasonTask
+import org.gradle.kotlin.dsl.support.appendReproducibleNewLine
+
+internal class ModuleAdviceExplainer(
+  private val project: ProjectCoordinates,
+  private val unfilteredAndroidScore: AndroidScore?,
+  private val finalAndroidScore: AndroidScore?,
+) : ReasonTask.Explainer {
+
+  override fun computeReason() = buildString {
```

Comment:
Wow, today I learned about the method `buildString` 😱 .  I just used `StringBuilder().apply{}` before.